### ISSUE TITLE
js can't execute right when I run parseInt("08")

### ISSUE THREAD

**Issue by BobJackson** [wkhtmltopdf/wkhtmltopdf#3803]
parseInt("08") = 0 !? parseInt("09") =0 !? 
I think it's  so amzing ! I have to handle the bug !

**Comment by alloylab** [comment]
What version?

**Comment by BobJackson** [comment]
wkhtmltopdf 0.12.4 (with patched qt)

**Comment by BobJackson** [comment]
Yes, I had tried on  0.13.0-alpha-7b36694 , parseInt("08") =8, thanks a lot.
However I want a stable version in production env, hope you make a success as soon as possible.

**Comment by grahampaulcook** [comment]
@BobJackson Is it within your code that parseInt("08") is being used? If so then there is a 2nd argument to parseInt that you can use to set the radix. You will want to set that to base 10 for decimal. Older versions of ECMS Script will default to octal base 8 and causes 8's and 9's to be 0 on calling parseInt as 8's and 9's do not exist in Octal.
